overthebox/defaults/network.defaults

```diff
@@ -10,6 +10,17 @@ if [ $? -ne 0 ]; then
 EOF
 fi
 
+MACPREFIX="42:42:42"
+if ! uci -q get network.lan.macaddr 2>/dev/null; then
+	MACSUFFIX=$(hexdump -n3 -e'3/1 ":%02x"' /dev/urandom)
```

Comment:
We already had issues with reads of /dev/urandom
We should always double check if it's ok or not